shell/platform/fuchsia/flutter/tests/integration/embedder/flutter-embedder-test.cc

```diff
@@ -160,29 +156,6 @@ class FlutterEmbedderTest : public ::loop_fixture::RealLoop,
           callback = nullptr,
       zx::duration timeout = kTestTimeout);
 
-  // Simulates a tap at location (x, y).
-  void InjectTap(int32_t x, int32_t y);
-
-  // Injects an input event, and posts a task to retry after
-  // `kTapRetryInterval`.
-  //
-  // We post the retry task because the first input event we send to Flutter may
-  // be lost. The reason the first event may be lost is that there is a race
-  // condition as the scene owner starts up.
-  //
-  // More specifically: in order for our app
-  // to receive the injected input, two things must be true before we inject
-  // touch input:
-  // * The Scenic root view must have been installed, and
-  // * The Input Pipeline must have received a viewport to inject touch into.
-  //
-  // The problem we have is that the `is_rendering` signal that we monitor only
-  // guarantees us the view is ready. If the viewport is not ready in Input
-  // Pipeline at that time, it will drop the touch event.
-  //
-  // TODO(fxbug.dev/96986): Improve synchronization and remove retry logic.
-  void TryInject(int32_t x, int32_t y);
```

Comment:
Can we also remove `RegisterTouchScreen` below?

Reply:
Removed!